ipa-core/src/protocol/basics/check_zero.rs

```diff
@@ -61,7 +63,7 @@ where
             .expect("full reveal should always return a value"),
     );
 
-    Ok(rv == F::ZERO)
+    Ok(rv.ct_eq(&F::ZERO).into())
```

Comment:
In `Fp32BitPrime` this isn't going to matter, but we're using MACs in `Fp25519` now.